host/xtest/regression_4000.c

```diff
@@ -4457,7 +4457,7 @@ static bool test_dh_key_pair(ADBG_Case_t *c, TEEC_Session *s,
 		KEY_ATTR(TEE_ATTR_DH_PRIME, false),
 		KEY_ATTR(TEE_ATTR_DH_BASE, false),
 		KEY_ATTR(TEE_ATTR_DH_PUBLIC_VALUE, false),
-		KEY_ATTR(TEE_ATTR_DH_PRIVATE_VALUE, check_keysize),
+		KEY_ATTR(TEE_ATTR_DH_PRIVATE_VALUE, false),
```

Comment:
Would it be possible to do something clever in `test_keygen_attributes()` to see if it's a black key and then allow for a larger size? Anything is better than nothing.